platform/android/java/app/config.gradle

```diff
@@ -1,8 +1,8 @@
 ext.versions = [
-    androidGradlePlugin: '4.2.1',
-    compileSdk         : 29,
-    minSdk             : 18,
-    targetSdk          : 29,
+    androidGradlePlugin: '4.2.2',
+    compileSdk         : 30,
+    minSdk             : 19,
```

Comment:
What's the reason for the minSdk bump? (Not against it, just curious.)

Reply:
Ah I see it's in the commit title:

> In addition, we're also bumping the min sdk version to 19 as version 18 is no longer supported by Google Play Services and only account of 0.3% of Android devices.

BTW, commit messages should usually be formatted like this:
```
Short title summarizing the change

Extended description paragraph(s).
```

The current commit has everything in the title since there's no separation line:
```
[3.3-scoped-storage 2c1f1dfc47] Add support for Android scoped storage. This is done by restricting read/write operations to the app specific directories. With that out of the way, we're also able to bump the target sdk version to 30. In addition, we're also bumping the min sdk version to 19 as version 18 is no longer supported by Google Play Services and only account of 0.3% of Android devices.
```

Reply:
I've updated the commit message to match the format!